### DISCUSSION TITLE
Spec: Help me to understand transfers

### DISCUSSION THREAD

**Discussion by dexX7** [ordinals/ord#1618] [Q&A] [answered]
Hello all,

I may be missing something very significant, but as far as I understand, a Satoshi is assigned an ordinal number when it's mined.

Does it's ordinal number change, when it's transferred, or do we have to trace back transactions to it's origin of mining?

**Comment by casey** [comment, marked as answer]
When a satoshi is mined, it receives an ordinal number. It keeps that ordinal number throughout all future transactions. Feel free to ask follow-up questions!
